scripts/prettify_components.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Often I use `#!/bin/bash -e` as shebang, because the default behaviour to continue the script after an error is usually quite undesired.

Reply:
added 👍